app/Http/Middleware/UpdateUserInfo.php

```diff
@@ -42,6 +42,13 @@ public function handle($request, Closure $next)
                         $user->update([
                             'user_lastvisit' => $currentLastVisit,
                         ], ['skipValidations' => true]);
+
+                        if ($token !== null) {
```

Comment:
What are the cases where token is non-null here? Just thinking about the `count` column – we'd probably want to increment it, but not if it's going to increment too often.

Reply:
when it's a password client token

Reply:
on every request?

Reply:
Hmm, might be okay to update count if it's debounced to 5 minutes.